src/ansys/fluent/core/solver/allowed_name_error_msg.py

```diff
@@ -19,3 +19,9 @@ def allowed_name_error_message(
     if matches:
         message += f"The most similar names are: {', '.join(matches)}."
     return message
+
```

Comment:
@hpohekar, as discussed, we don't want to have this file in here. We can simply add a file allowed_name_error_message.py with single line:
```
from ..allowed_name_error_message import allowed_name_error_message
```
In both versions of flobject.py:
```
from .allowed_name_error_message import allowed_name_error_message
```
We need t ou update the filename also, as mentioned just now.

Reply:
How about, add this line in src\ansys\fluent\core\solver\__init__.py:
```
from ansys.fluent.core import allowed_name_error_message
```
In flobject.py:
```
from allowed_name_error_message import allowed_name_error_message, closest_allowed_names
```
Add allowed_name_error_message.py at the same level of flobject.py in Fluent.

Reply:
@mkundu1 I tried that version first but got an import error (partial / cyclic import)

Reply:
Yeah, this won't work, we need to create the file.

Reply:
Done. Thank you.